contracts/tests/test_validator_auction.py

```diff
@@ -346,7 +360,7 @@ def test_event_whitelist(no_whitelist_validator_auction_contract, whitelist, web
 
 
 ETH_IN_WEI = 1e18
-START_PRICE = 10000 * ETH_IN_WEI
+AUCTION_START_PRICE = 10000 * ETH_IN_WEI
```

Comment:
Should this not be at the beginning of the file?
Also you have duplication of `AUCTION_START_PRICE`

Reply:
Fixed